### ISSUE TITLE
Script-ELM and armproject bug

### ISSUE THREAD

**Issue by zohebnsr** [ruffle-rs/ruffle#13014]
### Describe the bug

I am getting this error on console while trying few games:

Refused to oad the script 'nttos:/www.papasgaming.com/ruffle-update/ruffle.is because it violates the following Content Security Policy directive: "script-src https://cdn
ampproject.org/". Note that 'script-src-elem' was not explicitly set, so 'script-src' is us
ed as atal back.

Is it a ruffle bug or something in my end?

The game loads, but it stops in the middle showing a blank screen. It happens on Windows and Linux. Works fine on Mac, though.

### Expected behavior

It was working fine ealier.

### Affected platform

Self-hosted version

### Operating system

Windows 11, Linux, Windows 10

### Browser

Google Chrome, Edge

**Comment by Lord-McSweeney** [comment]
This is an issue with the site's Content Security Policy. They need to allow Ruffle to be loaded.

**Comment by zohebnsr** [comment]
Ruffle is locally hosted and running Ngnix.
Never to allow ?

**Comment by zohebnsr** [comment]
> This is an issue with the site's Content Security Policy. They need to allow Ruffle to be loaded.

There is something else I believe. The content loads, but it gets issue in the middle after some steps.

Can you test this please and suggest?: https://www.papasgaming.com/unblocked/test-post/300852784/

**Comment by n0samu** [comment]
Closing in favor of #13020
